uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/BAO/LeaveRequestTest.php

```diff
@@ -1388,4 +1511,9 @@ public function testLeaveRequestCanNotBeCreatedWhenTheDatesOverlapMoreThanOneCon
       'to_date_type' => $toType
     ]);
   }
+
+  private function registerCurrentLoggedInContactInSession($contactID) {
```

Comment:
This piece of code is starting to get duplicated among a few different test classes. What about extracting it to a trait?